.github/workflows/build-and-test-v3.yml

```
@@ -0,0 +1,247 @@
+name: Build + Test v3
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+    branches:
+      - v3-alpha
+    paths:
+      - 'v3/**'
+  pull_request_review:
+    types: [submitted]
+    branches:
```

actionlint (1.7.7)

"branches" filter is not available for pull_request_review event. it is only for merge_group, push, pull_request, pull_request_target, workflow_run events

(events)